akka-actor-tests/src/test/scala/akka/io/TcpConnectionSpec.scala

```diff
@@ -364,7 +364,7 @@ class TcpConnectionSpec extends AkkaSpec("""
         ConfigFactory.load(
           ConfigFactory.parseString("akka.io.tcp.direct-buffer-size = 1k")
             .withFallback(AkkaSpec.testConf))
-      override implicit def system: ActorSystem = ActorSystem("respectPullModeTest", config)
+      override lazy val system: ActorSystem = ActorSystem("respectPullModeTest", config)
```

Comment:
The failure seems to indicate that the config has been not applied. I.e. not the right ActorSystem is used. Maybe multiple systems are needed here?